client/src/components/ProfileSelection/ProfileSelection.tsx

```diff
@@ -0,0 +1,25 @@
+import useStyles from './useStyles';
+import Grid from '@material-ui/core/Grid';
+import { Link } from 'react-router-dom';
+import Button from '@material-ui/core/Button';
+
+const ChatSideBanner = (props: { name: string; link: string; selected: boolean }): JSX.Element => {
```

Comment:
```suggestion
const SidebarLink= (props: { name: string; link: string; selected: boolean }): JSX.Element => {
```

Why don't we give this component a more appropriate name 😄